README.md

```diff
@@ -122,17 +128,17 @@ Most of the implementation is a direct translation from [turf.js](https://github
 - [ ] coordAll
 - [ ] coordEach
 - [ ] coordReduce
-- [ ] featureEach
+- [x] featureEach
```

Comment:
should we link to the source code?

Reply:
fixed, but with line numbers. I'm not convinced yet if that's a good idea. It works now, but might break later

Reply:
I agree that it will break in the future. Maybe we can later link to the API reference on pub.dev https://pub.dev/documentation/turf/latest/